### ISSUE TITLE
QT_PLUGINS_PATH system variable issue

### ISSUE THREAD

**Issue by IWhoI** [OpenShot/openshot-qt#1213]
**System Details:**
Windows 10
Uninstalled it before checking the version of OpenShot

OpenShot sets a system variable QT_PLUGINS_PATH that apparently prevents OpenVR Advanced Settings and Revive from loading images in the SteamVR Dashboard. The errors in the ReviveOverlay log are like this:
qrc:/Overlay.qml:84:13: QML Image: Error decoding: qrc:/SupportAssets/oculus-dreamdeck-nux/cover_square_image.jpg: Unsupported image format
Uninstalling OpenShot does not remove the system variable and it has to be manually deleted to get the images to show in the SteamVR Dashboard.
Link to Revive issue report: https://github.com/LibreVR/Revive/issues/389

**Comment by DylanC** [comment]
@IWhoI - Thanks!

**Comment by steel3d** [comment]
@DylanC Is this ever gonna get fixed? You guys are breaking all qt apps.

**Comment by SuslikV** [comment]
@steel3d where did you find it? What installation/version was used?

**Comment by steel3d** [comment]
@SuslikV sorry I missed the question back in the day. It was a customer's machine, don't know what the install was, where it came from and why. The question is: are you guys still setting this environment variable? If you are, you're breaking other QT apps.

**Comment by ferdnyc** [comment]
@steel3d 

Sorry for the glacial pace of things here, I only just saw this go by in my email notifications.

The answer is, we are no longer setting `QT_PLUGIN_PATH` anywhere except as a local envvar in one of our launch scripts. (Using just standard `SET QT_PLUGIN_PATH=...` Windows batch commands, no registry-fiddling.)

The ill-advised code to forcibly _set_ `QT_PLUGIN_PATH` in the system environment was removed in 2016. But then after that, for a long-long time we were forcibly **removing** the key during install. (Under the guise of cleaning up after ourselves; feels like somebody didn't really think that one through.) I finally put a stop to that nonsense with 8dbbf54ad91004e36d65227c23064e2f747e777a on 2019-07-14.
